src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -159,7 +204,24 @@ export const WorkspaceDashboard = _.flow(
             ])
           ],
           [!!description, () => h(Markdown, [description])],
-          () => div({ style: { fontStyle: 'italic' } }, ['No description added']))
+          () => div({ style: { fontStyle: 'italic' } }, ['No description added'])),
+        _.some(_.startsWith('library:'), _.keys(attributes)) && h(Fragment, [
+          div({ style: styles.header }, ['Dataset Attributes']),
+          h(SimpleTable, {
+            rows: _.flow(
+              _.map(({ key, title }) => ({ name: title, value: displayAttributeValue(attributes[key]) })),
+              Utils.append({
+                name: 'Structured Data Use Limitations',
+                value: attributes['library:orsp'] ? consentStatus : h(DataUseLimitations, { attributes })
```

Comment:
For those that have blank values, can we remove them so they don't clutter the list?

Reply:
Thanks, filtered out the empty values and increased the spacing between items.

Reply:
Do we know if users will understand what each of these codes mean?

Reply:
You can mouse over the abbreviations for a full description, just like in FC.

Reply:
OH I see now! 
My concern is that this behavior is pretty different from how we normally deliver information. My first thought would be that it would be in a tooltip, rather than a mouse-hover over. It's a lot of text for a tool tip, so there's a tradeoff. 

That being said, I think it may be telling that I didn't know I needed to mouse over and hold for a bit before the description came up until you just told me about it.

Reply:
Fair point, changed to use TooltipTrigger.